Formula/atmos.rb

```diff
@@ -0,0 +1,97 @@
+class Atmos < Formula
+  desc "Universal Tool for DevOps and Cloud Automation"
+  homepage "https://github.com/cloudposse/atmos"
+  url "https://github.com/cloudposse/atmos/archive/refs/tags/v1.3.9.tar.gz"
+  sha256 "dabae8c89e508bd150640ec562f2642854c50a8e6ffbfd881a701ae40394f533"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOARCH"] = Hardware::CPU.intel? ? "amd64" : Hardware::CPU.arch.to_s
+    ENV["GOOS"] = OS.kernel_name.downcase
+    system "go", "build", *std_go_args(ldflags: "-X 'github.com/cloudposse/atmos/cmd.Version=#{version}'")
```

Comment:
we dont need to specify GOARCH or GOOS, as there is no cross-build happening with homebrew process
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w -X 'github.com/cloudposse/atmos/cmd.Version=#{version}'")
```

Reply:
@chenrui333 don't we have to adjust for linuxbrew users? or does it automatically set the correct environment variable when on linux or osx?

Reply:
there are also other formula that use GOOS and GOARCH env vars

https://github.com/Homebrew/homebrew-core/search?q=goos&type=

Reply:
@chenrui333 I made the above changes and force pushed.